.gitignore

```diff
@@ -1 +1,2 @@
 /bin/
+/auth/
```

Comment:
> I've added /auth/ to gitignore as well, though it looks like there's a preference not to include any of the other assets that may be generated when running the installer inside a checkout. Let me know if you'd like for me to remove that.

Yeah, probably remove this based on previous discussion in #969 (which discusses `/auth` explicitly) and #996 (which clarifies the policy as "we don't have `.gitignore` entries for asset-directory content").